<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stacked_widget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>541</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>541</width>
     <height>261</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>541</width>
     <height>261</height>
    </size>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="admin_newpwd_le">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin_surepwd_le">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>84</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin_id_le">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>36</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>84</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="admin_ok_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确   认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin_oldpwd_le">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="admin_back_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>返    回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>编  号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>旧密码：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>书    名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_bookname_le">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学    号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_stuid_le">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="out_ok_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>导    出</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="out_book_acount">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>48</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>图书编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_bookid_le">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学        号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_stuid_le">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="in_ok_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>导    入</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="in_book_acount">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>48</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_stuid_le">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_stuname_le">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学院编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_depid_le">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>专业编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_speid_le">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_sex_le">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_initpwd_le">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="create_btn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>创    建</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学        号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="revise_stuid_le">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="revise_newpwd_le2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="revise_btn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>修    改</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="revise_newpwd_le">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>新 密 码 ：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学  号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="delete_stuid_le">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>姓  名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="delete_stuname_le">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>删    除</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>201</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>修改密码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书籍借出</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书籍导入</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>创建用户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>修改用户密码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>删除用户</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
